diff --git a/packages/cli/src/services/frontend.service.ts b/packages/cli/src/services/frontend.service.ts
index 18d220269f..6e9227e9dd 100644
--- a/packages/cli/src/services/frontend.service.ts
+++ b/packages/cli/src/services/frontend.service.ts
@@ -331,16 +331,16 @@ export class FrontendService {

 			aiBuilder: {
-				enabled: false,
-				setup: false,
+				enabled: true,
+				setup: true,
 			},
 			aiCredits: {
-				enabled: false,
-				credits: 0,
-				setup: false,
+				enabled: true,
+				credits: -1,
+				setup: true,

